jetstream/metadata.py

```diff
@@ -19,10 +19,18 @@ class MetricsMetadata:
     bigger_is_better: bool
 
 
+@attr.s(auto_attribs=True)
+class OutcomeMetadata:
+    friendly_name: str
+    description: str
+    metrics: List[str]
+
+
```

Comment:
Does this seem like useful information to be part of the metadata?

Reply:
Yeah, I think so. Can we include the slug as well?

@emtwo Is there anything else you'd want in the metadata associated with an experiment to describe an outcome?